src/core/init.c

```diff
@@ -211,7 +211,7 @@ struct ap_section ap_sects[] = {
 struct ap_settings ap_sets = {
 	"ROOT-Sim",	// TODO properly fill these fields
 	"Proper version string",
```

Comment:
This should be populated with the version in meson build files.

Reply:
Good call! This will be fixed in the next commit.